var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -178,6 +178,9 @@ def libs(self):
     conflicts('+taggedlayout', when='+versionedlayout')
     conflicts('+numpy', when='~python')
 
+    # boost-python in 1.72.0 broken with cxxstd=98
+    conflicts('cxxstd=98', when='+mpi+python @1.72.0:')
```

Comment:
So it's not broken with `+mpi~python`, `~mpi+python`, or any other possible combo? Just making sure.

Reply:
No, only if both are enabled.